UnityClient/Assets/Scripts/CameraSwitching.cs

```diff
@@ -4,20 +4,19 @@
 public class CameraSwitching : MonoBehaviour
 {
     [SerializeField]
-    private Camera _mainCamera;
+    private GameObject _cameraChoice;
 
     [SerializeField]
-    private Camera _overheadCamera;
+    private Transform _allCameras;
```

Comment:
Possibly not for this PR I don't think, but this might be better off as a `Camera[]` or a `List<Camera>`, as the current setup creates an expectation of the heirarchy. Its also partially hiding the actual dependency/dependencies.